modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/ProcessInstanceCollectionResourceTest.java

```diff
@@ -466,55 +483,61 @@ public void testStartProcessWithVariablesAndReturnVariables() throws Exception {
+                        + "               name: 'integerVariable',"
+                        + "               value: 1234,"
+                        + "               type: 'integer'"
+                        + "              } ]"
+                        + "}");
 
         ProcessInstance processInstance = runtimeService.createProcessInstanceQuery().singleResult();
-        assertNotNull(processInstance);
+        assertThat(processInstance).isNotNull();
 
         // Check if engine has correct variables set
         Map<String, Object> processVariables = runtimeService.getVariables(processInstance.getId());
-        assertEquals(2, processVariables.size());
+        assertThat(processVariables).hasSize(2);
 
-        assertEquals("simple string value", processVariables.get("stringVariable"));
-        assertEquals(1234, processVariables.get("integerVariable"));
+        assertThat(processVariables.get("stringVariable")).isEqualTo("simple string value");
+        assertThat(processVariables.get("integerVariable")).isEqualTo(1234);
     }
-    
+
     @Test
     @Deployment(resources = { "org/flowable/rest/service/api/runtime/ProcessInstanceResourceTest.process-with-form.bpmn20.xml",
-                    "org/flowable/rest/service/api/runtime/simple.form"})
+            "org/flowable/rest/service/api/runtime/simple.form" })
     public void testStartProcessWithForm() throws Exception {
         ProcessDefinition processDefinition = repositoryService.createProcessDefinitionQuery().processDefinitionKey("processOne").singleResult();
         try {
             FormDefinition formDefinition = formRepositoryService.createFormDefinitionQuery().formDefinitionKey("form1").singleResult();
-            assertNotNull(formDefinition);
-            
+            assertThat(formDefinition).isNotNull();
+
             FormInstance formInstance = formEngineFormService.createFormInstanceQuery().formDefinitionId(formDefinition.getId()).singleResult();
-            assertNull(formInstance);
-            
+            assertThat(formInstance).isNull();
+
             String url = RestUrls.createRelativeResourceUrl(RestUrls.URL_PROCESS_DEFINITION_START_FORM, processDefinition.getId());
             CloseableHttpResponse response = executeRequest(new HttpGet(SERVER_URL_PREFIX + url), HttpStatus.SC_OK);
             JsonNode responseNode = objectMapper.readTree(response.getEntity().getContent());
             closeResponse(response);
-            assertEquals(formDefinition.getId(), responseNode.get("id").asText());
-            assertEquals(formDefinition.getKey(), responseNode.get("key").asText());
-            assertEquals(formDefinition.getName(), responseNode.get("name").asText());
-            assertEquals(2, responseNode.get("fields").size());
-            
+            assertThatJson(responseNode)
+                    .when(Option.IGNORING_EXTRA_FIELDS, Option.IGNORING_EXTRA_ARRAY_ITEMS, Option.IGNORING_ARRAY_ORDER)
```

Comment:
The `IGNORING_EXTRA_ARRAY_ITEMS` is extra here. We should verify that there are only 2 elements. Sorry I didn't see this before

Reply:
No problem; fixed.  Thanks again.